services/src/main/java/com/linkedin/kmf/services/TopicManagementService.java

```diff
@@ -391,7 +391,7 @@ public boolean isRunning() {
   }
 
   @Override
-  public synchronized void awaitShutdown() {
+  public void awaitShutdown() {
```

Comment:
Note that start() and close() are also marked with keyward `synchronized`. An althernative, which is the pattern used in other service class, is to not have keyword `synchronized` and make `_running` a `AtomaticBoolean`. Would that be better?